libs/enums.py

```diff
@@ -4,20 +4,26 @@
 # TODO: This should maybe be unique per state.
 UNKNOWN_FIPS = "99999"
 
+
+class AggregateLevel(enum.Enum):
+    STATE = 0
+    COUNTY = 1
```

Comment:
Hmm is there a reason that we should have both `AggregationLevel` (see dataset/dataset_utils.py) and `AggregateLevel`? It'd be great to combine those two.

Reply:
ahh cool thought they were class vars.